src/function/loadEmojione.js

```diff
@@ -33,49 +41,90 @@ function(emojione, uniRegexp, emojioneVersion, readyCallbacks, emojioneSupportMo
+                    readyCallbacks.shift().call();
+                }
+                cdn.isLoading = false;
+            });
+        }
+
+        options = getOptions(options);
         if (!cdn.isLoading) {
             if (!emojione || getSupportMode(detectVersion(emojione)) < 2) {
                 cdn.isLoading = true;
-                $.getScript(cdn.defaultBase + emojioneVersion + "/lib/js/emojione.min.js", function () {
-                    emojione = window.emojione;
-                    emojioneVersion = detectVersion(emojione);
-                    emojioneSupportMode = getSupportMode(emojioneVersion);
-                    cdn.base = cdn.defaultBase + emojioneVersion + "/assets";
-                    if (options.sprite) {
-                        var sprite = cdn.base + "/sprites/emojione.sprites.css";
-                        if (document.createStyleSheet) {
-                            document.createStyleSheet(sprite);
-                        } else {
-                            $('<link/>', {rel: 'stylesheet', href: sprite}).appendTo('head');
-                        }
-                    }
-                    while (readyCallbacks.length) {
-                        readyCallbacks.shift().call();
-                    }
-                    cdn.isLoading = false;
-                });
+                if(getSupportMode(emojioneVersion) > 4) {
+                    loadEmojione3Lib();
+                } else {
+                    loadEmojione2Lib();
+                }
             } else {
                 emojioneVersion = detectVersion(emojione);
                 emojioneSupportMode = getSupportMode(emojioneVersion);
-                cdn.base = cdn.defaultBase + emojioneVersion + "/assets";
+                if(emojioneSupportMode > 4) {
+                    cdn.base = cdn.defaultBase3 + "-assets@" + emojioneVersion;
+                } else {
+                    cdn.base = cdn.defaultBase + emojioneVersion + "/assets";
+                }
             }
         }
 
         emojioneReady(function() {
+            var emojiSize = "";
+
             if (options.useInternalCDN) {
-                emojione.imagePathPNG = cdn.base + "/png/";
-                emojione.imagePathSVG = cdn.base + "/svg/";
+                if(emojioneSupportMode > 4) emojiSize = emojione.emojiSize + "/";
+                
+                emojione.imagePathPNG = cdn.base + "/png/" + emojiSize;
+                emojione.imagePathSVG = cdn.base + "/svg/" + emojiSize;
                 emojione.imagePathSVGSprites = cdn.base + "/sprites/emojione.sprites.svg";
                 emojione.imageType = options.imageType;
             }
-
-            uniRegexp = new RegExp("<object[^>]*>.*?<\/object>|<span[^>]*>.*?<\/span>|<(?:object|embed|svg|img|div|span|p|a)[^>]*>|(" + emojione.unicodeRegexp + ")", "gi");
+            if(getSupportMode(emojioneVersion) > 4) {
+                uniRegexp = new RegExp("(" + emojione.regUnicode + ")", "gi");
```

Comment:
`<img src="..." alt="😄"/>` will transform to `<img src="..." alt="<img src="..." alt="😄"/>"/>`

Reply:
It did go wrong. My idea was to use the same regex of emojione 3 ("uniRegexp = emojione.regUnicode;") but I forgot to change the code.
One observation is that the regex of emojione 3 in javascript has some problems. There are a few issues opened about that on git.